transport-classes-io_uring/src/main/java/io/netty/channel/uring/IoUring.java

```diff
@@ -188,6 +191,15 @@ public final class IoUring {
         IORING_POLL_ADD_MULTISHOT_ENABLED = IORING_POLL_ADD_MULTISHOT_SUPPORTED && SystemPropertyUtil.getBoolean(
                "io.netty.iouring.pollAddMultishotEnabled", true);
         NUM_ELEMENTS_IOVEC = numElementsIoVec;
+
+        DEFAULT_RING_SIZE =  Math.max(64, SystemPropertyUtil.getInt("io.netty.iouring.ringSize", 128));
```

Comment:
64 seems like a high minimum now, though it's unchanged from before.

Reply:
yeah maybe lets do 16 ...